src/Backend/Form/Type/TagsType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Backend\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class TagsType extends AbstractType
+{
+    public function configureOptions(OptionsResolver $optionsResolver): void
+    {
+        $optionsResolver->setDefaults(
+            [
+                'attr' => [
+                    'class' => 'js-tags-input',
+                    'aria-describedby' => 'tags-info',
+                ],
+                'label' => 'lbl.Tags',
+            ]
+        );
+    }
+
+    public function getParent(): string
+    {
+        return TextType::class;
+    }
+
+    public function getBlockPrefix(): string
+    {
+        return 'tags';
```

Comment:
This is probably the default block prefix so it can be safely removed

Reply:
fixed in ada6900dd